.ci/appveyor.yml

```diff
@@ -51,6 +51,10 @@ install:
 build: false  # Not a C# project, build stuff at the test step instead.
 
 test_script:
+  # Force DOS format, as Checkstyle configs enable NewlineAtEndOfFile,
+  # which defaults to CRLF on Windows, and Appveyor CI ignores .gitattributes
+  # http://help.appveyor.com/discussions/problems/5687-gitattributes-changes-dont-have-any-effect
+  - unix2dos tests/java/test_files/CheckstyleGood.java
```

Comment:
I dont object to doing that; certainly makes sense.

However pushing back a little bit, and only a little bit, this line fixes one problem occurring in `test_style_android`, and it occurs only on Windows.  The bad files already fail, for other reasons, and fail on unix, so the only thing which more `unix2dos` would find is more Windows specific issues wrt checkstyle.

When I upgraded to Checkstyle 7.6 (https://github.com/coala/coala-bears/pull/1466), there are three failed method - `test_style_sun` and `test_style_google` also fail [on Appveyor](https://ci.appveyor.com/project/coala/coala-bears/build/1.0.5492/job/jx37mkvuaa0faidm) and not on Travis/Circle , so there are more Windows specific problems somewhere in there, which will be found in the next set of patches for CheckstyleBear.